### PR TITLE
fix(secrets): change default value of secret values to empty strings

### PR DESCRIPTION
**Pull request by marynaKK**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/secrets/runner.py [hunk 1]
```diff
@@ -202,16 +202,16 @@ def run(
...
                 if runner_filter.enable_git_history_secret_scan:
                     enriched_potential_secret = git_history_scanner.\
                         history_store.get_added_and_removed_commit_hash(key, secret, root_folder)
-                    added_commit_hash = enriched_potential_secret.get('added_commit_hash')
-                    removed_commit_hash = enriched_potential_secret.get('removed_commit_hash')
-                    code_line = enriched_potential_secret.get('code_line')
-                    added_by = enriched_potential_secret.get('added_by')
-                    removed_date = enriched_potential_secret.get('removed_date')
-                    added_date = enriched_potential_secret.get('added_date')
+                    added_commit_hash = enriched_potential_secret.get('added_commit_hash') or ''
```
Comment (pazbechor) [inline checkov/secrets/runner.py:209]:
```suggestion
                    added_commit_hash = enriched_potential_secret.get('added_commit_hash') if enriched_potential_secret.get('added_commit_hash') else ''
```

**Review by pazbechor — APPROVED** [review]
lgtm

File: checkov/secrets/runner.py (hunk 1 above)
Comment (gruebel) [reply checkov/secrets/runner.py:209]: @marynaKK original code does exactly this, no need to change it

**Review by gruebel — APPROVED** [review]
🍪

**Pull request merged**